Escape library as a table name in tests (#12170)

mbeccati opened the pull request:
It is a reserved word since MySQL 9.2:
https://dev.mysql.com/doc/refman/9.2/en/create-library.html

Fixes #12170

greg0ire commented on tests/Tests/Models/Enums/Library.php:
```diff
@@ -11,8 +11,10 @@
+use Doctrine\ORM\Mapping\Table;
 
 #[Entity]
+#[Table('library1')]
```
Would the following fix the issue?

```suggestion
#[Table('`library`')]
```

[The docs](https://www.doctrine-project.org/projects/doctrine-orm/en/3.5/reference/basic-mapping.html#quoting-reserved-words) suggest it would.

mbeccati replied:
Yes, indeed.
I had seen backticks used in definitions, but I always thought it was because the underlying platform was MySQL, as double quotes are the SQL standard for quoting identifiers. Thanks!

mbeccati committed:
Avoid using LIBRARY as table name (#12170)

It is a reserved word since MySQL 9.2:
https://dev.mysql.com/doc/refman/9.2/en/create-library.html
